saleor/graphql/product/mutations/products.py

```diff
@@ -268,13 +278,20 @@ def user_is_allowed(cls, user, input):
 
 
 class ProductVariantInput(graphene.InputObjectType):
-    attributes = graphene.List(AttributeValueInput)
-    cost_price = Decimal()
-    price_override = Decimal()
-    product = graphene.ID()
-    sku = graphene.String()
-    quantity = graphene.Int()
-    track_inventory = graphene.Boolean(required=True)
+    attributes = graphene.List(
+        AttributeValueInput,
+        description='List of product attributes.')
+    cost_price = Decimal(description='Cost price of the variant.')
+    price_override = Decimal(
+        description='Special price of the particular variant.')
+    product = graphene.ID(
+        description='Product ID of which type is the variant.')
+    sku = graphene.String(description='SKU.')
+    quantity = graphene.Int(
+        description='Available amount of products of this variant.')
+    track_inventory = graphene.Boolean(
+        required=True,
+        description='Determines if inventoryof the variant should be tracked.')
```

Comment:
Determines if the inventory of this variant should be tracked. If `false`, the quantity won't change when customers buy this item.